test/fixtures/tree/mixed-indentation.commonmark.json

```diff
@@ -53,7 +53,7 @@
               "children": [
                 {
                   "type": "text",
-                  "value": "Very long\n\t\t\tparagraph",
+                  "value": "Very long\n\tparagraph",
```

Comment:
This comes from trying to find the indentation of 
```
- Very long
<tab><tab>paragraph
```
This text is the "internal" content, the content without the indentation.

The resulting should be 
```
Very long
<tab>paragraph
```

as the `- ` in the first line marks the indentation, that is matched by the first `<tab>` in the second line.

Making this changes in the fixture makes it more consistent with the rest of the rest of the `mixed-indentation` fixtures, in all flavours and list styles.